RecoTracker/MeasurementDet/src/TkMeasurementDetSet.h

```diff
@@ -389,6 +402,7 @@ class PxMeasurementDetSet {
   std::vector<PixelDetSet> detSet_;
   std::vector<bool> empty_;
   std::vector<bool> activeThisEvent_;
+  std::vector<std::unique_ptr<BadFEDChannelPositions> > badFEDChannelPositionsSet_;
```

Comment:
how is this better than a vector of vectors?

Reply:
avoids constructing a lot of empty vectors for detId-s with no bad channels

Reply:
a map would be the more natural data structure given the code used to fill this.